File: python/class2.py
class dog:
    def __init__(self,name,size):
        self.name = name
        self.age = 5
        self.color = "brown"
        self.size = size

    def bark(self):
        if self.size == "small":
            return "yip yip"
        elif self.size == "medium":
            return "woof woof"
        elif self.size == "large":
            return "WOOF WOOF"

maya = dog("maya", "small")


print(maya.bark())
print(maya.name)

File: Java/class2/class2.java
class dog {

    int age;

    dog(int a) {
        age = a;
    }

    void bark() {
        if (age > 5) {
            IO.println("Woof");
        } else if (age > 2) {
            IO.println("Ruff");
        } else {
            IO.println("Yip");
        }
    }

}

void main() {
    dog d1 = new dog(3);
    dog d2 = new dog(6);
    d1.bark();
    d2.bark();

}